apps/editing-toolkit/editing-toolkit-plugin/newspack-blocks/carousel-block-editor.js

```diff
@@ -1,14 +1,14 @@
 import { registerBlockType } from '@wordpress/blocks';
 import { addFilter } from '@wordpress/hooks';
+import { CAROUSEL_BLOCK_NAME } from './consts';
 import { settings } from './synced-newspack-blocks/blocks/carousel/index';
 
 /**
- * Block name in the A8C\FSE context.
+ * Internal dependencies
  */
```

Comment:
I think this entire comment could be deleted. At least, "internal dependencies" seems out of place :)

Reply:
Removed in a1caf1c. Thanks for the catch!